libsolidity/interface/DebugSettings.h

```diff
@@ -0,0 +1,63 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * Settings to aid debugging.
+ */
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <optional>
+
+namespace dev
+{
+namespace solidity
+{
+
+enum class RevertStrings
+{
+
+	Default, // no compiler-generated strings, keep user-supplied strings
+	Strip, // no compiler-generated strings, remove user-supplied strings (if possible)
+	Debug, // add strings for internal reverts, keep user-supplied strings
+	VerboseDebug // add strings for internal reverts, add user-supplied strings if not provided
+};
+
+inline std::string revertStringsToString(RevertStrings _str)
```

Comment:
I think, usually such conversion functions that convert to string are called `to_string(...)`. I know we have no common guidelines for that (and we should modernize ours maybe), but I'd at least state, that having a little consistency would probably help maintenance a bit.

Reply:
I used `to_string` before but it resulted in compilation errors elsewhere. I have not tried putting it in the global namespace, though.